corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -54,6 +56,26 @@ hqDefine('accounting/js/pricing_table', function () {
             });
         };
 
+        self.updateIsAnnualPricing = function () {
+            self.isAnnualPricing(true);
+            console.log(self.isAnnualPricing())
+        };
+        self.updateIsMonthlyPricing = function () {
+            self.isAnnualPricing(false);
+            console.log(self.isAnnualPricing())
+        };
+
+        self.openContactSalesModal = function (pricingTable, e) {
```

Comment:
You can open a modal without writing your own js. In the HTML, in the button that should open the modal, add the attributes `data-toggle="modal"` and `href="#myModalId"`, then give your modal (the outermost div, with the `modal` class) the id `myModalId`.

This works via code in `bootstrap.js` that scans the page for elements with `data-toggle="modal"` and hooks them up with modals.

Reply:
There's also a `modal` binding that might be useful here, which opens a modal depending on the state of some knockout observable. The [filter panel in UCR](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/templates/userreports/partials/filter_panel.html#L53) does this - the button to open the modal runs a function when it's clicked, and that function sets a value, which triggers the modal to open.

Reply:
...having read fully through this commit now, I think it'd likely simplify the code to do one modal instead of two, and update its content, replacing the form with the thank you message, once they send the email. Sorry about the multiple comments.

Reply:
No worries! I am getting rid of one of the modals anyway, so this PR is only getting more confusing. I'm getting rid of the "Contact Sales/Thank you" modal, as that will just be its own page.
For the "Downgrading?" modal, how do I tell knockout to only show the modal if the user is downgrading, and to wait to send the form data until the user has clicked OK on the modal?